src/verify/test_friendship.sql

```diff
@@ -0,0 +1,78 @@
+BEGIN;
+
+DO $$
+DECLARE
+  accountA UUID;
+  accountB UUID;
+  accountC UUID;
+
+  friendshipAB UUID;
+  friendshipAC UUID;
+  friendshipCA UUID;
+
+BEGIN
+
+  -- fill with test data
+
+  accountA := maevsi_test.friendship_account_create('a', 'a@example.com');
+  accountB := maevsi_test.friendship_account_create('b', 'b@example.com');
+  accountC := maevsi_test.friendship_account_create('c', 'c@example.com');
+
+  RAISE NOTICE 'accountA = %', accountA;
+  RAISE NOTICE 'accountB = %', accountB;
+  RAISE NOTICE 'accountC = %', accountC;
+
+  -- run tests
+
+  RAISE NOTICE 'test 1';
+
+  friendshipAB := maevsi_test.friendship_request(accountA, accountB);
+  RAISE NOTICE 'friendshipAB = %', friendshipAB;
+
+  PERFORM maevsi_test.friendship_test ('1a', accountA, 'pending', ARRAY[friendshipAB]::UUID[]);
```

Comment:
```suggestion
  PERFORM maevsi_test.friendship_test('The friendship is pending for user A', accountA, 'pending', ARRAY[friendshipAB]::UUID[]);
```

We can describe the test case here and this line will show up in the stacktrace if the test fails, so we don't need to keep the `RAISE NOTICE`s. I'll write test descriptions for all test cases. You can resolve this comment if you're fine with the change.